From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Javier Martinez Canillas <javierm@redhat.com>
Date: Wed, 11 Apr 2018 11:36:43 +0200
Subject: [PATCH] Add missing options to grub2-switch-to-blscfg man page

The script --bls-directory and --backup-suffix options were not documented
in the man page, add them as well so users can know what these are about.

Signed-off-by: Javier Martinez Canillas <javierm@redhat.com>
---
 util/grub-switch-to-blscfg.8 | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/util/grub-switch-to-blscfg.8 b/util/grub-switch-to-blscfg.8
index 134dfc62a7b..7d99de2d2f9 100644
--- a/util/grub-switch-to-blscfg.8
+++ b/util/grub-switch-to-blscfg.8
@@ -21,5 +21,13 @@ The grub config file to use.  The default value is \fI/etc/grub2-efi.cfg\fR on U
 --grub-defaults=\fIFILE\fR
 The defaults file for grub-mkconfig.  The default value is \fI/etc/default/grub\fR.
 
+.TP
+--bls-directory=\fIDIR\fR
+Create BootLoaderSpec fragments in \fIDIR\fR.  The default value is \fI/boot/loader/entries\fR on BIOS machines and \fI/boot/efi/EFI/\fBVENDOR\fI/loader/entries\fR on UEFI machines.
+
+.TP
+--backup-suffix=\fSUFFIX\fR
+The suffix to use for saved backup files.  The default value is \fI.bak\fR.
+
 .SH SEE ALSO
 .BR "info grub"
